### ISSUE TITLE
yarn tauri init fails on Ubuntu with (node:581958) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '1' of null

### ISSUE THREAD

**Issue by happybeing** [tauri-apps/tauri#1037]
**Describe the bug**
I'm setting up on Ubuntu 20.04 and already had Rust/Cargo/Yarn/nvm installed. 

Following the getting started for Linux at tauri.studio all is well until `yarn tauri init` which gives the following error. I wonder if it is to do with node version as I've seen things fail at this point, but tauri didn't specify which version it uses. So I tried node v10 and v12, both fail with:

```
$ yarn tauri init 
yarn run v1.22.4
$ /home/mrh/src/tauri/tauri-test/node_modules/.bin/tauri init
[tauri]: running init
? What is your app name? Tauri Test
? What should the window title be? Tauri Test
? What is the url of your dev server? http://localhost:4000
? Where are your web assets (HTML/CSS/JS) located, relative to the "<current dir>/src-tauri" folder
 that will be created? ../dist
 dependency:manager Installing missing dependencies... +0ms
 dependency:cargo-commands "tauri-bundler" is already installed +26ms
 app:spawn [sync] Running "cargo generate-lockfile" +3ms

    Updating crates.io index

 dependency:crates "tauri" is already installed +474ms
(node:583252) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '1' of null
    at /home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:984:57
    at new Promise (<anonymous>)
    at Object.<anonymous> (/home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:982:46)
    at step (/home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:932:23)
    at Object.next (/home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:913:53)
    at /home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:907:71
    at new Promise (<anonymous>)
    at ./src/api/dependency-manager/util.ts.__awaiter (/home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:903:12)
    at Object.getNpmPackageVersion (/home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:979:12)
    at /home/mrh/src/tauri/tauri-test/node_modules/tauri/dist/api/dependency-manager.js:654:49
(node:583252) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:583252) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 13.98s.
```

**To Reproduce**
```
sudo apt update && sudo apt install libwebkit2gtk-4.0-dev \
    build-essential \
    curl \
    libssl-dev \
    appmenu-gtk3-module \
    libgtk-3-dev
rustup update stable
cargo install tauri-bundler --force
mkdir tauri-test
cd tauri-test
yarn init
yarn add tauri
yarn tauri init
```

**Platform and Versions (please complete the following information):**

OS: Ubunto 20.04
Node: v12.16.3 (same result with v10.19.0)
NPM: n/a
Yarn: 1.22.4
Rustc: 1.46.0

**Comment by happybeing** [comment]
I didn't have `npm` installed and had skipped the following because I already had set up `node` with `nvm`. When I inspected the line where the error is I saw it using `npm`. The following has fixed this issue:
```
nvm install node --latest-npm
nvm use node
```

I think this is an edge case few will hit, but given the Tauri team ecourage people to use `yarn`, someone like myself who already prefers `yarn` may well not have `npm` installed, particularly on a recently set-up machine. That was my issue.

So maybe tauri init should check for existance of programs it uses such as `npm`? I'll leave this open to register that question, but feel free to close.
